src/table/table.scss

```diff
@@ -122,265 +249,341 @@
-            }
-          }
+      &:hover {
+        @include themed {
+          background: rgba(
+            t(iui-color-foreground-primary-rgb),
+            t(iui-opacity-5)
+          );
         }
+      }
 
-        > .iui-filter,
-        > .iui-sort,
-        > .iui-sort > .iui-icon-wrapper {
-          height: $iui-icons-default;
-          width: $iui-icons-default;
-          display: flex;
-          justify-content: center;
-          align-items: center;
-          visibility: hidden;
+      + .iui-selected {
+        border-bottom: 0;
+        padding-bottom: $iui-xxs;
+        padding-top: 0;
+        @include themed {
+          border-top: 1px solid
+            rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-4));
         }
+      }
 
-        > .iui-filter.iui-active {
-          visibility: visible;
-  
-          > .iui-icon {
-            @include themed {
-              fill: t(iui-icons-color-primary);
-            }
-          }
+      &:last-child {
+        padding-bottom: 1px;
+        @include themed {
+          border-bottom: 1px solid t(iui-color-foreground-primary);
         }
+      }
 
-        &:hover {
-          @include themed {
-            background-color: t(iui-color-background-4);
-          }
-
-          > .iui-filter,
-          > .iui-sort,
-          > .iui-sort > .iui-icon-wrapper {
-            visibility: visible;
-          }
-        }
+      &:first-child {
+        border-bottom: 0;
+        padding-bottom: $iui-xxs;
       }
 
-      &.iui-active-sort {
+      // Last selected item in a row.
+      + :not(.iui-selected) {
+        padding-top: 0;
+        margin-left: 0;
+        margin-right: 0;
         @include themed {
-          background-color: t(iui-color-background-4);
+          border-top: 1px solid t(iui-color-foreground-primary);
         }
+      }
 
-        > .iui-sort,
-        > .iui-sort > .iui-icon-wrapper {
-          visibility: visible;
+      // Added by Jon, not fully tested
+      + .iui-expanded-content {
+        @include themed {
+          border-color: t(iui-color-foreground-primary);
         }
       }
     }
 
-    &.iui-tables-row-active {
-      @include themed {
-        border: solid 1px t(iui-color-foreground-primary);
-        background: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
-      }
+    &:last-child {
+      padding-bottom: $iui-xxs;
+      border-bottom: 0;
     }
+    // #endregion Selection
 
-    &.iui-tables-row-new {
+    // #region Statuses & states
+    &.iui-new {
       font-weight: $iui-font-weight-semibold;
-
-      > .iui-tables-filename {
-        position: relative;
-      }
-
-      > .iui-tables-filename::before {
+      
+      > .iui-main-column::before {
         @include themed {
           color: t(iui-color-foreground-positive);
         }
         content: '•';
         position: absolute;
         left: 0;
         font-size: $iui-font-size-title;
-      }
+      }   
```

Comment:
Remove extra space here 😮

Reply:
Is prettier set ?

Reply:
Prettier is not used for html files anymore. We switched to default vsc formatter (which should be able to handle this).

https://github.com/iTwin/iTwinUI/blob/3fb2a6bd60eb936a1d79b8c8adf24ded0f6391c0/.vscode/settings.json#L18-L19

Reply:
Right, my b. In that case we switched from prettier to stylelint.
https://github.com/iTwin/iTwinUI/blob/3fb2a6bd60eb936a1d79b8c8adf24ded0f6391c0/.vscode/settings.json#L6

Reply:
Ok, I think I removed the trailing spaces.